### PR TITLE
BAU: Add pull request template

### PR DESCRIPTION
**Pull request by merlinc**
This is a PR template from a previous project to use as a discussion point.

As we're in alpha and exploring, I've removed sections relating to screenshots, unit testing and change logs. They can be restored at a more appropriate time.

When we're happy with the structure, we should duplicate it across our repositories, and add it into the handbook as an example configuration.

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!-- Provide a general summary of your changes in the Title above -->
+<!-- Include the Jira ticket number in square brackets as prefix, eg `[P4-XXXX] PR Title` -->
```
Comment (merlinc) [inline pull_request_template.md]:
```suggestion
<!-- Include the Jira ticket number in square brackets as prefix, eg `[PYI-XXXX] PR Title` -->
```

**Review by AmritSidhu — DISMISSED** [review]
👍🏽

**Comment by merlinc** [comment]
@kerrr @martin-bucinskas Any thoughts before I merge this, and roll it out to our other repositories?

**Review by martin-bucinskas — DISMISSED** [review]
> @kerrr @martin-bucinskas Any thoughts before I merge this, and roll it out to our other repositories?

Looks good to me

**Review by AmritSidhu — DISMISSED** [review]

**Review by AmritSidhu — APPROVED** [review]

**Pull request merged**